### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by NOVA-ME**
Signed-off-by: Ivan Zhou <nova-me@outlook.com>

## What have you changed? (mandatory)

image
- Change the CONTRIBUTING.md's style doc url to [https://doc.rust-lang.org/1.0.0/style/](https://doc.rust-lang.org/1.0.0/style/), the old code style website return the 404.

## What are the type of the changes? (mandatory)

- Improve the document

## How has this PR been tested? (mandatory)

- Click the 'style doc', it will go to the rust official document(The Rust Style Guidelines).:smile:

## Does this PR affect documentation (docs) or release note? (mandatory)

- No

## Does this PR affect tidb-ansible update? (mandatory)

- No

### CODE REVIEW

**Comment by ice1000** [comment]
Missing sign-offs

**Comment by ice1000** [comment]
You need to signoff your old commits.

**Comment by NOVA-ME** [comment]
> You need to signoff your old commits.

I'm not good at git. I use the github desktop push to the repository,Can I use git rebase -i <commit range> to make the three commit to one?

**Comment by siddontang** [comment]
Thanks @NOVA-ME 

Please sign off your all commits. See https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md#signing-off-the-commit

If you have already pushed some commits before, you can use `git rebase -i master --signoff` to resign them off.

**Comment by NOVA-ME** [comment]
> Thanks @NOVA-ME
> 
> Please sign off your all commits. See https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md#signing-off-the-commit
> 
> If you have already pushed some commits before, you can use `git rebase -i master --signoff` to resign them off.

Thank you very much. I've been learned so many git command today.:small:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Thanks for your contributions!
 
 ### Code style
 
-The coding style suggested by the rust community and clippy project. See the [style doc](https://aturon.github.io/README.html) and [clippy](https://github.com/Manishearth/rust-clippy) for details.
+The coding style suggested by the rust community and clippy project. See the [style doc](https://doc.rust-lang.org/1.0.0/style/README.html) and [clippy](https://github.com/Manishearth/rust-clippy) for details.
```
Comment (BusyJay) [inline CONTRIBUTING.md]: Now official guide becomes https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md.

Comment (NOVA-ME) [reply CONTRIBUTING.md]: > Now official guide becomes https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md.

Thank you. I‘ve been learn rust about a week, this guide very useful to me.

Comment (siddontang) [reply CONTRIBUTING.md]: should we update to the new official guide here? @BusyJay

Comment (BusyJay) [reply CONTRIBUTING.md]: Yes, please update it with the provided link.

Comment (siddontang) [reply CONTRIBUTING.md]: Friendly ping @NOVA-ME

Comment (NOVA-ME) [reply CONTRIBUTING.md]: > Friendly ping @NOVA-ME

Everything is ok. I'm sry reply so late. A little busy these days.

**Comment by brson** [comment]
Thanks @NOVA-ME !

**Pull request merged**